Switcher/TargetInformationBuilder.php

```diff
@@ -31,17 +33,22 @@ class TargetInformationBuilder
     private $allowedLocales;
 
     /**
-     * @param Request         $request           Request
-     * @param RouterInterface $router            Router
-     * @param array           $allowedLocales    Config Var
-     * @param bool            $showCurrentLocale Config Var
-     * @param bool            $useController     Config Var
+     * @param RequestStack $requestStack Request
+     * @param RouterInterface $router Router
+     * @param AllowedLocalesProvider $allowedLocalesProvider
+     * @param bool $showCurrentLocale Config Var
+     * @param bool $useController Config Var
```

Comment:
does this repo not align parameters?

Reply:
i don't know what the style guide of this repo is.

Reply:
looking at random files, it seems to align them, so i suggest you keep them aligned in the files you touched.

@lunetics can you please enable styleci for this repo?